giraffe/src/utils/formatters.ts

```diff
@@ -139,6 +139,7 @@ export const timeFormatter = ({
   // for example, modifying the timezone before determing the `am/pm` will
   // output the timezone incorrectly. The same goes for determining the `HH`, etc...
   const formatStringFormatter = createDateFormatter({
+    hh: ({hour}) => (Number(hour) < 10 ? `0${hour}` : String(hour)),
```

Comment:
If we're looking to prepend a `0` we can use the `lhour` prop since that should reflect the `24h` hour

Reply:
Prepending 0 is to ensure that hour displays correctly in all environments. Locally, I was seeing "4" for four o'clock, but in CircleCI it was "04", causing tests to pass locally but fail in the pipeline.

Using lhour doesn't solve this problem. Four o'clock PM becomes "16", subtracting 12 gives "4" -- which requires the 0 prepend.

Which brings up a good point: there is now a subtle difference between using `hh:mm a` versus `HH:mm a` -- the leading zero may or may not be there. `hh:mm a` will always display the leading zero. So, now we have an extra option for more subtleties in the formatting.

Reply:
👍